common/middleware/ensure-body-processed.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function ensureBodyProcessed(authMountpoint = '/auth') {
```

Comment:
Could this be tied more to the auth processing? Maybe it's just a lib/function we call once when that happens. 

Doesn't feel like its a global level middleware we want to be running on every request (including things like assets). We know the specific point in time and case it needs to happen, so we can run it just the once then.

Reply:
It's not running on every request (certainly not on assets) as it is called directly before the main router is.

I argue that it is the specific time that it needs to run - ie. the request has been authenticated and we're about to have HMPO form wizard (via our router) handle the request. That is the exact point at which we need to update the req values.

Reply:
Sorry, I meant this middleware is being run on every request as it is attached to the global app router.

It feels heavy to have this check run on every request when we know at what point we need to copy across the `originalRequestBody` to `req.body`. Could we do it as part of the auth middleware callback?

Reply:
Discussed